css/custom.css

```diff
@@ -964,3 +964,19 @@ h1 .tm, h2 .tm, h3 .tm {
     color: #555;
     font-weight: 500;
 }
+
+/* GitHub style blockquote */
+blockquote {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 40px;
+    margin-inline-end: 40px;
+}
+
+blockquote {
```

Comment:
Merge this into line 975?

Reply:
Merging it removes
![image (1)](https://user-images.githubusercontent.com/6477701/168200850-ef65564d-2082-4d94-ab09-12d827ce6b4a.png) for some reasons. Both are like that in GitHub CSS anyway. should be fine to leave it as is.